xml/containers-docker-dockerizing-application.xml

```diff
@@ -63,23 +57,12 @@
     You can prepare an image of the complete build environment.
    </para>
   </listitem>
-  <listitem>
-   <para>
-    The build always runs in the same environment that the application is
```

Comment:
This can actually be kept, but would need a fix like `The application can run in the same environment it was built in`.
Though I'm wondering if there's overlap with the next item.